.travis.yml

```diff
@@ -1,13 +1,25 @@
 language: node_js
 
+# https://docs.travis-ci.com/user/build-stages/deploy-npm/
 node_js:
-  - 8.9.4
+  - "8.9.4"
 
-install:
-  - npm install
-  - npm install -g vsce
+jobs:
+  include:
+    - stage: TS Linter
+      script:
+        - echo "Run TS linter ..."
+        - npm install tslint typescript -g
```

Comment:
Unnecessary install-- these packages are declared as devDependencies and thus already available. Call `tslint` either via `npx` or by adding this command to `scripts` in package.json.